tensornetwork/backends/numpy/numpy_backend.py

```diff
@@ -169,10 +171,9 @@ def eigs(self,
            maxiter: Optional[int] = None) -> Tuple[Tensor, List]:
     """
     Arnoldi method for finding the lowest eigenvector-eigenvalue pairs
-    of a linear operator `A`. `A` can be either a
-    scipy.sparse.linalg.LinearOperator object or a regular callable.
-    If no `initial_state` is provided then `A` has to have an attribute
-    `shape` so that a suitable initial state can be randomly generated.
+    of a linear operator `A`. If no `initial_state` is provided then 
+    `shape` and `dtype` are required so that a suitable initial state can be 
```

Comment:
Do you think "state" is the appropriate term here? It seems specific to DMRG

Reply:
Let's leave it for now because it's like that through all backends. Happy to change it in another PR